payload-receiver-service/tests/readme.md

```diff
@@ -0,0 +1,26 @@
+# Validation scripts for Payload Receiver Service
+Validation tests are run in a similar way to integration tests.
+
+## From tests/validation dir:
+
+Setup a virtual environment and save the kubeconfig into the kube.yaml file
+
+pip3 install -r requirements.txt
+
+## Export any needed environment vars:
+
+export NATS_SERVER_URL=nats://localhost:4222
+export NATS_USERNAME=nats_client
```

Comment:
Remove the NATS_USERNAME and NATS_PASSWORD and instead have the user specify the NKEY_SEED_FILENAME